custom-ports/ffmpeg/0001-create-lib-libraries.patch

```diff
@@ -0,0 +1,25 @@
+diff --git a/configure b/configure
+index 34c2adb..1c8008a 100755
+--- a/configure
++++ b/configure
+@@ -5631,15 +5631,17 @@ case $target_os in
+         ;;
+     win32|win64)
+         disable symver
+-        if enabled shared; then
++#        if enabled shared; then
```

Comment:
Oh, we had to modify the ffmpeg port to build on WIndows?

Reply:
This commit was a direct copy from the original port with no changes, and then I put my own commits on top of it.  The only things I modified were things in portfile.cmake.

Reply:
I have consolidated the adding of the port and the customizing of the port down to these two commits:
- 5f0e8cd338fff9568a5104c5559ecb0c9c349d9a
- d840ce323d7a73879fb2c3dcadbf3a76fc1dd480

Also, I put a comment above and below the section where we are customizing it, to make it easy to identify.